docker/depends/pecan.depends

```diff
@@ -17,7 +18,7 @@ installGithub.r \
     ropensci/nneo
 
 # install all packages (depends, imports, suggests)
-install2.r -e -s -n -1\
+install2.r -e -s -l "${RLIB}" -n -1\
```

Comment:
Attn @robkooper: Am I right this shouldn't change any behavior inside Docker? Added so that we can reuse the script for Travis caching.

Reply:
I think this should be ok.